src/core/Akka/Util/WildcardIndex.cs

```diff
@@ -7,21 +7,21 @@ namespace Akka.Util
 {
     internal sealed class WildcardIndex<T> : IEquatable<WildcardIndex<T>> where T : class
     {
-        private readonly WildcardTree<T> _wildcardTree;
-        private readonly WildcardTree<T> _doubleWildcardTree;
+        public WildcardTree<T> WildcardTree { get; }
```

Comment:
Is this used for something inside Artery?

Reply:
yeah, they're accessed directly from one of the classes. I forgot which one...

Reply:
Got it.